nifi-nar-bundles/nifi-kudu-bundle/nifi-kudu-processors/src/test/java/org/apache/nifi/processors/kudu/TestPutKudu.java

```diff
@@ -521,11 +560,13 @@ private void testKuduPartialFailure(FlushMode flushMode, int batchSize) throws E
         testRunner.setProperty(PutKudu.FLUSH_MODE, flushMode.name());
         testRunner.setProperty(PutKudu.BATCH_SIZE, String.valueOf(batchSize));
         testRunner.setProperty(PutKudu.FLOWFILE_BATCH_SIZE, String.valueOf(batchSize));
+        testRunner.setValidateExpressionUsage(false);
```

Comment:
Why is it required to add this? I usually try avoiding it.

Reply:
@pvillard31 without setting setValidateExpressionUsage(false), we see below error for all test cases

```
Attempting to evaluate expression language for Table Name without using flow file attributes but the scope evaluation is set to FLOWFILE_ATTRIBUTES.
```

Reply:
IIRC, I think this shouldn't be necessary once you remove the table name evaluation in the onScheduled method